hack/verify-flags/known-flags.txt

```diff
@@ -393,6 +393,7 @@ max-pods
 max-requests-inflight
 maximum-dead-containers
 maximum-dead-containers-per-container
+message
```

Comment:
I'd like move this under https://github.com/kubernetes/kubernetes/blob/master/hack/verify-flags/known-flags.txt#L410 to keep alpha order.

```
mesos-user
message
min-available
```